scripts/updates/utils/termux_pkg_upgrade_version.sh

```diff
@@ -96,8 +96,10 @@ termux_pkg_upgrade_version() {
 		fi
 	done
 
-	# Get available space (df outputs sizes in 1024-byte blocks)
-	local space_available="$(df -P "/var/lib/docker" | awk 'NR==2 {print $4}')"
+	local cleanup_threshold="$(( 5 * 1024 ** 3 ))" # 3 GiB
+
+	# Get available space in bytes
+	local space_available="$(df "/var/lib/docker" | awk 'NR==2 { val=$4*1024 } END { if(!val) val='$cleanup_threshold'+1; print val }')"
```

Comment:
You cannot expect `/var/lib/docker` to present in all configurations. It can be changed. I strongly oppose fallback to hardcoded threshold+1 value.

Reply:
Ok, fixed this.